identity/app/controllers/UpsellController.scala

```diff
@@ -50,8 +50,10 @@ class UpsellController(
     authenticatedActions.consentAuthWithIdapiUserAction.async { implicit request =>
       val returnUrl = returnUrlVerifier.getVerifiedReturnUrl(request)
       val email = request.user.primaryEmailAddress
+      val hasPassword = request.user.hasPassword.getOrElse(true)
```

Comment:
would it be ok putting a comment here as to why we are defaulting to `true`? In this context should the `Option` ever be `None`?